FluentAssertions.Core/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -101,7 +101,7 @@ public class EquivalencyAssertionOptions : EquivalencyAssertionOptionsBase<Equiv
         public EquivalencyAssertionOptions()
         {
             IncludingNestedObjects();
-            IncludingAllDeclaredProperties();
+            IncludingAllDeclaredMembers();
```

Comment:
No, and I think that would be a serious breaking change. Fields should represent internal state, so including them in the assertion would only make sense in exceptional situations,